site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -287,13 +288,13 @@ These examples require the following host permission:
 To work out the priority of a particular URL, look at the (developer-defined) `"priority"` key, the `"action"` key and the `"urlFilter"` key. These examples refer to the example rule file shown below them.
 
 **Navigation to https://google.com**
-Two rules cover this URL: the rules with IDs 1 and 4. The rule with ID 1 applies because `"block"` actions have a higher priority than `"redirect"` actions. The remaining rules do not apply because they are for longer URLs.
+: Two rules cover this URL: the rules with IDs 1 and 4. The rule with ID 1 applies because `"block"` actions have a higher priority than `"redirect"` actions. The remaining rules do not apply because they are for longer URLs.
```

Comment:
There are now two subsequent colons in this paragraph

Reply:
This is how that renders:
![image](https://github.com/GoogleChrome/developer.chrome.com/assets/8204171/e16a1fe9-5c05-41ad-80e6-0f15d9efb0af)

Reply:
an nice, all good then